src/query_parser/v2/lexer.rl

```diff
@@ -29,9 +29,9 @@ void RSQuery_ParseFree_v2(void *p, void (*freeProc)(void *));
 
 machine query;
 
-inf = ['+\-']? 'inf' $ 4;
+inf = ('+'|'-')? 'inf'i $ 4;
```

Comment:
Consider leaving this as it is, it's quite nice :)

Reply:
I mean the `['+\-']` only

Reply:
BTW not sure that we need the quotes here.

Reply:
OK. Change done. Thanks.